### PR TITLE
remove some deprecation warnings

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

File: Tests/WebTest/TestApp/FileTest.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ public function setUp()
 
     public function testPage()
     {
-        $client = $this->createClient();
+        $client = $this->getClient();
```
Comment (dbu) [inline Tests/WebTest/TestApp/FileTest.php]: this got me rid of the symfony 2.8 failures.

**Comment by dbu** [comment]
@WouterJ is there a way to only see the deprecations about stuff inside this bundle, and not vendors? i ran the tests with `SYMFONY_DEPRECATIONS_HELPER=strict` and found some stuff, but now it seems most of the rest comes from jms serializer and other 3rd party things.

**Comment by WouterJ** [comment]
@dbu sorry for ignoring this (I decided to click on "mark all notifications as read" a couple days ago...). No, it is not possible to show only deprecations of this bundle. I'll have a look at the remaining notices now

**Comment by dbu** [comment]
thanks. worst case we can simply switch back to `weak` and ignore the
warnings for now, as things will work on symfony 2.8 - just not on 3.0

**Comment by WouterJ** [comment]
Fixed some more deprecations and turned the mode to weak again. I don't think we should put more time into Symfony 3 support for 1.3 atm.

**Comment by dbu** [comment]
thanks!

**Pull request merged**